metrics-next/src/metrics.rs

```diff
@@ -0,0 +1,29 @@
+use {
+    crate::core,
+    prometheus_client::{encoding::text::encode, registry::Registry},
+};
+
+pub struct Metrics {
+    pub registry: Registry,
+    pub core: core::Metrics,
```

Comment:
To confirm - the idea would be to have one of these per crate? Or, we could even hypothetically break it up like
- `core::BankingStageMetrics`
- `core::ReplayStageMetrics`
- ...

Reply:
I don't have a strong feeling here but the crate group is the plan in my mind. just would like have a place to manage all datapoints.